docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'qkay'
+copyright = '2024, esavary'
```

Comment:
The copyright should be assigned to nipreps, as all others:

```suggestion
copyright = '2024 The NiPreps Developers'
```

Reply:
Yes, thanks for catching that. Sorry, my GitHub handle was added when I generated the template with Sphinx. Since you've started a review, do you have any additional comments, or is the rest ready to merge?